packages/graph/hash_graph/bench/benches/representative_read/mod.rs

```diff
@@ -65,11 +65,6 @@ fn bench_representative_read_multiple_entities(c: &mut Criterion) {
     let (runtime, store_wrapper) = setup(DB_NAME, false, false);
     group.sample_size(10);
     group.sampling_mode(SamplingMode::Flat);
-    // Set the warm_up and measurement times to try and encourage Criterion to run as few iterations
-    // as possible as these benchmarks run a *lot* longer than the intended Criterion benchmark,
-    // by a few orders of magnitude
-    group.warm_up_time(Duration::from_nanos(1));
```

Comment:
Why are you removing these?

Reply:
I aligned the benchmarks to be the same as in `main`, so I removed them locally, they were not supposed to be removed in the branch. However, I noticed more reliable time measurements with the warm-up time to be the default value (3 seconds) so probably it's worth leaving them out for now.

Reply:
They'll be _slightly_ more reliable, but in my experience they run a lot longer because Criterion picks more iterations 🤷, I don't feel strongly though, resolving.